src/bash_src/modeling/mergeEstimation/mergeEstimation_help.txt

```diff
@@ -5,12 +5,12 @@ Usage:
     mergeEstimation.sh [options] -- MAT_FILES PREFIX CHR_NUM TOTAL_PROC
 
 Options:
-    -h|--help           Help.
-    -r|--refdir         Reference genome directory. Default: $REFGENEDIR.
-    -m|--matdir         Matrices directory. Default: $INTERDIR.
-    -e|--estdir         Estimation directory. Default: $INTERDIR.
-    -d|--outdir         Output directory. Default: $INTERDIR.
-    -l|--MATLICENSE     Path to MATLAB's License. Default: <NA>.
+    -h|--help           help.
+    -r|--refdir         reference genome directory (default: $REFGENEDIR)
+    -m|--matdir         matrices directory (default: $INTERDIR)
+    -e|--estdir         estimation directory (default: $INTERDIR)
+    -d|--outdir         output directory (default: $INTERDIR)
+    -l|--MATLICENSE     path to MATLAB's License (default: <NA>)
 
 Examples:
     mergeEstimation.sh -d out -- normal-1 normal-1 1 1
```

Comment:
Just a thought: what if all our examples all excluded the "--"? This makes the commands shorter and more in-line with other tools, and we would be less likely to miss the mistake we discussed previously where "--" is interpreted as an argument to an optional argument flag. If people do not use the "--" it seems more likely to me that we catch problems in argument parsing right away.

Reply:
Yes, I agree. The following would not throw an error early on:
 `command.sh -d -- a`
But the following would throw an error right away:
 `command.sh -d a`
So yes, I will go ahead and remove them all from the examples.

Reply:
Taken care of.